.github/new_pipeline/pull_request_template.md

```diff
@@ -0,0 +1,71 @@
+---
+name: New pipeline
+description: Initiate the peer review process for a new pipeline
+title: "[NEW PIPELINE]: "
+labels: ["pipeline", "user contributed", "peer review needed"]
+---
+
+> [!IMPORTANT]
+> To facilitate the peer review of the new pipeline, do not change the structure of this
+> document. Only the parts in comments should be replaced.
+
+## General information about the pipeline
+
+**Title:** <!-- TITLE GOES HERE -->
+
+| Author | Affiliation | GitHub ID | ORCID | [CRediT Role][role] |
+|-------|-----|----|----|---|
+| | | | | |
+
+[role]: https://credit.niso.org/
+
+> [!INFO]
+> Abstract of the pipeline goes here
+
+## Code information
+
+**GitHub repo:** <!-- REPO URL -->
+
+**LICENSE:** <!-- LICENSE NAME HERE -->
+
+> [!NOTE]
+> The pipeline license **must** be FOSS, **should** not require the same license for derived
+> products, and **must** be in either `LICENSE` or `LICENSE.md` in the folder of the
+> package. You can delete this note when done.
+
+**Languages used:**
+
+- [ ] R (version)
+- [ ] Julia (version)
+- [ ] Python (version)
+- [ ] other
+
+> [!INFO]
+> For each *other* language, copy this quote block and list the language, version, and other
+> relevant information
+
+**Dependencies manager:** <!-- LIST HERE -->
+
+> [!INFO]
+> Pipelines **must** ship with a list of their dependencies, including information about
+> which versions are usable. You can delete this note when done.
+
+## Additional information
+
+**Testing:** <!-- free-form text to explain the testing/CI of the pipeline -->
+
+## Optional (recommended)
+
+- [ ] [Mermaid][mermaid] diagram
+
+[mermaid]: https://github.blog/developer-skills/github/include-diagrams-markdown-files-mermaid/
+
+## Review information
+
+| Date | Step | Comments | User |
+|----|----|----|----|
+| <!-- TODAY YYYY-MM-DD --> | Submission | | <!-- YOUR GITHUB ID --> |
+| | Initial check | | |
+| | Review started | | |
+| | Reviewer 1 invited | | |
+| | Reviewer 2 invited | | |
```

Comment:
Most review information and comments should be captured by the history of the PR. Maybe replace this table with a "progress of PR checklist" ?

Progress of Pull Request (to be filled by reviewers):
- [ ] Initial check by <!-- name -->
- [ ] Open review call started <!-- time span -->
- [ ] Reviewers invited, <!-- names -->
- [ ] Review 1 completed by <!-- name -->
- [ ] Review 2 completed by <!-- name -->
- [ ] Modifications received
- [ ] Modifications validated
- [ ] Merged
- [ ] Published on the web